cnxarchive/database.py

```diff
@@ -60,19 +64,33 @@ def _read_sql_file(name):
 
 
 def initdb(settings):
-    """Initialize the database from the given settings."""
-    connection_string = settings[config.CONNECTION_STRING]
+    """Initialize the database from the given settings.
+       If a superuser is given in the settings then the
+       function will use the superuser to initialize the
+       database.
+    """
+    if settings['super-connection-string']:
+        connection_string = settings['super-connection-string']
+    else:
+        connection_string = settings['db-connection-string']
```

Comment:
I was wondering about this.  Why are we using ``config.CONNECTION_STRING``?  Is there a situation where the dictionary keys for accessing our connection strings change?